bigquery/snippets/querying/bigquery_query.go

```diff
@@ -40,19 +40,11 @@ func queryBasic(w io.Writer, projectID string) error {
 			"LIMIT 100")
 	// Location must match that of the dataset(s) referenced in the query.
 	q.Location = "US"
-	// Run the query and print results when the query job is completed.
-	job, err := q.Run(ctx)
+	// Run the query and process the returned row iterator.
+	it, err := q.Read(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("query.Read(: %v", err)
```

Comment:
assuming these should all be `"query.Read(): %v` (closing the parens)?

Reply:
There is a saying among my people to express my feelings here:

"D'oh!"